hello-nextflow/hello-config.nf

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env nextflow
+// Include modules
```

Comment:
Is the idea here that module includes should go before anything else? Is there a functional reason or is it purely stylistic? Any interactions wrt setting any variable values?

Reply:
In principle, it's just a convention as it shouldn't matter where you put the includes

In practice, users sometimes put the params first so that they are propagated to the included modules as a side effect. We are trying to move away from that pattern anyway